src/main/kotlin/org/dashj/platform/dpp/statetransition/StateTransition.kt

```diff
@@ -30,8 +30,7 @@ abstract class StateTransition(
         DOCUMENTS_BATCH(1),
         IDENTITY_CREATE(2),
         IDENTITY_TOP_UP(3),
-        IDENTITY_UPDATEKEY(4),
-        IDENTITY_CLOSEACCOUNT(5);
+        DATA_CONTRACT_UPDATE(4);
```

Comment:
These items may come back later, but there is no functionality for updating keys or closing accounts.
```kotlin
        IDENTITY_UPDATEKEY(4),
        IDENTITY_CLOSEACCOUNT(5);
```